src/explore-education-statistics-admin/src/pages/publication/__tests__/PublicationContactPage.test.tsx

```diff
@@ -173,10 +173,42 @@ describe('PublicationContactPage', () => {
         }),
       ).toBeInTheDocument();
 
+      // NOTE: Contact telephone number is optional, so no validation
+    });
+  });
+
+  test('show validation error when contact tel no is not valid', async () => {
```

Comment:
It would be a good idea to create more test cases for various permutations. This can be achieved succinctly by using `test.each` to provide test data e.g.

```ts
test.each([
  [' 03700002288 '],
  ['03700002288'],
  ['0370 000 2288'],
  [' 0 3 7 0 0 0 0 2 2 8 8 '],
])(
  'show validation error when contact tel no (%s) is DfE enquiries number',
  async telNo => {
    // the test case...
  }
);
```

Would suggest we also create test cases for:

- `show validation error when contact tel no is less than 8 characters`
- `show validation error when contact tel no is not only numeric or whitespace characters` (use `test.each` for multiple permutations of this)